generators/mail/templates/common/manifest.xml

```diff
@@ -7,6 +7,7 @@
   <DefaultLocale>en-US</DefaultLocale>
   <DisplayName DefaultValue="<%= projectDisplayName %>" />
   <Description DefaultValue="[Outlook Add-in description]"/>
+  <SupportUrl DefaultValue="http://contoso.com/support" />
```

Comment:
Great point! I am putting this down as a task for the sprint. Will close this pr then :)